extra/man/alacritty.5.scd

```diff
@@ -33,6 +42,12 @@ This section documents the root level of the configuration file.
 		Linux/BSD/macOS: _$SHELL_ or the user's login shell, if _$SHELL_ is unset++
 Windows: _"powershell"_
 
+	*Example:*
+
+		*[shell]*++
+program = _"/bin/bash"_++
+args = _["-l"]_
+
 *working_directory* <string> | "None"
```

Comment:
Is there a reason why we don't document examples for simple K/V pairs? Wouldn't it be better to have an example than the default, so you can copy-paste it?

Reply:
I wasn't sure how to document here because it's a global section.

Reply:
`working_directory = "whatever"`